BuildingBlocks/src/BuildingBlocks.Infrastructure/Persistence/BlobStorage/AzureStorageAccount/AzureStorageAccount.cs

```diff
@@ -78,7 +83,7 @@ public Task<IAsyncEnumerable<string>> FindAllAsync(string folder, string? prefix
 
     public async Task SaveAsync()
     {
-        await UploadChangedBlobs();
+        await _logger.TraceTime(async () => UploadChangedBlobs(), nameof(UploadChangedBlobs));
         await DeleteRemovedBlobs();
```

Comment:
For the sake of completeness, can you please add time logging for this call as well?

Reply:
Hmm, that is already done on my end... I'll check again once I'm done with all the other changes.

Reply:
And don't forget to do it for both implementations.